controllers/networking-calico/pkg/controller/actuator_reconcile.go

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controller
+
+import (
+	"context"
+
+	"github.com/gardener/gardener/pkg/operation/common"
+	"github.com/pkg/errors"
+
+	extensionscontroller "github.com/gardener/gardener-extensions/pkg/controller"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	resourcemanager "github.com/gardener/gardener-resource-manager/pkg/manager"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/gardener/gardener-extensions/controllers/networking-calico/pkg/charts"
+
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+)
+
+const (
+	calicoConfigSecretName = "extension-network-calico-config"
```

Comment:
Another inconsistent use of `network` vs `networking`

Reply:
I copied your suggestion for the secret name early on. Will change it.